src/SailthruChannel.php

```diff
@@ -67,6 +68,23 @@ public function send(
                 static::getDefaultVars()
             );
 
+            if (config('services.sailthru.whitelist.enabled') === true) {
```

Comment:
Is `config('services.sailthru.whitelist.enabled')` already enabled in the imo config?

Reply:
Also please consider calling it `whitelist_check` instead of just `whitelist` which is a bit confusing on the goal

Reply:
it will be enabled on imo and disabled on brk

Reply:
Would that be on a separate PR?

Reply:
you can set your whitelist to `['*']` on BRK and it will allow all addresses